src/main/java/rx/internal/operators/OperatorOnBackpressureBuffer.java

```diff
@@ -71,6 +175,7 @@ public OperatorOnBackpressureBuffer(long capacity, Action0 onOverflow) {
 
         return parent;
     }
+
     private static final class BufferSubscriber<T> extends Subscriber<T> implements BackpressureDrainManager.BackpressureQueueCallback {
```

Comment:
Thanks. I replaced with the parent interface implemented by BufferSubscriber, which is public.